src/driver_nodelet.cpp

```diff
@@ -42,6 +49,25 @@ void DriverNodelet::onInit() {
     set_up_default_transform();
   }
 
+
+///////////////////join mcast group/////////////
+
+    struct sockaddr_in localif_cepton;
+    struct ip_mreq mreq_cepton;
+    int s = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP);
+    localif_cepton.sin_family = AF_INET;
+    localif_cepton.sin_port   = htons(8808);
+    localif_cepton.sin_addr.s_addr = htonl(INADDR_ANY);
+    bind(s, (sockaddr *)&localif_cepton, sizeof(localif_cepton));
+    mreq_cepton.imr_interface.s_addr = inet_addr("192.168.0.100");
+   //mreq.imr_multiaddr.s_addr = inet_addr("232.79.85.83");
+    mreq_cepton.imr_multiaddr.s_addr = inet_addr("232.67.69.80");
```

Comment:
- Remove one of the redundant lines to set `s_addr`
- Instead of hard-coding the IP address, make it a ROS string parameter, similar to the `parent_frame_id` parameter on line 29.
- Remove commented line

Reply:
Oops, I didn't notice that you are setting `s_addr` for different interfaces. Please disregard. However, make both IP addresses separate ROS parameters